### DISCUSSION TITLE
Responsive

### DISCUSSION THREAD

**Discussion by OctavioBernalGH** [Power-Components/livewire-powergrid#1065] [BUG]
Good morning, I am trying to use the responsive class in powergrid but I get the error:

Class "PowerComponents\LivewirePowerGrid\Responsive" not found.

I have installed the latest version of PowerGrid for laravel and configured everything but I get this error, I am calling the responsive class in the function setUp()

public function setUp(): array
    {
        $this->showCheckBox();

        return [
            Exportable::make('export'))
                ->striped()
                ->type(Exportable::TYPE_XLS, Exportable::TYPE_CSV),
            Header::make()->showSearchInput(),
            Footer::make()
                ->showPerPage()
                ->showRecordCount(),
            Responsive::make()
        ];
    }

As indicated in the documentation.

**Comment by ashek1412** [comment]
any solution

**Reply by luanfreitasdev** [reply]
What version are you using? It seems to work normally. Have a look here:
https://github.com/Power-Components/powergrid-demo-misc/blob/main/app/Http/Livewire/DishesTable.php#L97

**Comment by ashek1412** [comment]
I updated to v4 and its working. But I cannot implement line break for long text column. is there any way to that like below,
![screeshot-title](https://github.com/Power-Components/livewire-powergrid/assets/18645594/748a5f21-7838-48e6-b2d5-5753327a56fe)
